src/app/state/staking/index.ts

```diff
@@ -4,11 +4,12 @@ import { useInjectReducer, useInjectSaga } from 'utils/redux-injectors'
 
 import { stakingSaga } from './saga'
 import { DebondingDelegation, Delegation, StakingState, Validator, ValidatorDetails } from './types'
+import * as validatorsDump from 'vendors/smartstake/validators_dump'
 
 export const initialState: StakingState = {
   debondingDelegations: [],
   delegations: [],
-  validators: [],
+  validators: validatorsDump.validators,
```

Comment:
So with this users can end up delegating to a stale validator that may no longer exist?  At a minimum staking operations while this data is being provided to users should be limited to debonding.

NACK.

Reply:
Fair point, agreed

Reply:
Mitigated by marking them all `inactive` for now, but I don't intend to merge ![image](https://user-images.githubusercontent.com/3758846/156863382-e8c01298-e635-4d4e-b309-a869d5264056.png)

Reply:
imo instead of changing initial state we should fallback in saga when a request fails. instead of dispatching `updateValidatorsError` action we can dispatch 
`yield* put(stakingActions.updateValidators(validatorsDump.validators))`

Reply:
One difference in that case (without extra handling): if getting validators first succeeds, but fails second time, result is:
- fallback on error: fallback
- fallback in initial state: first success

Reply:
ah true, is it possible that between the first and the second request validator could be updated? First success -> validator A works just fine -> something bad happens to validator A -> the second request to monitor fails  -> we show `fallback in initial state: first success` where validator A is Ok -> user delegates to a stale validator.

Reply:
Yeah, it is possible

Reply:
Added a warning before staking to validators with status inactive* or unknown:

![localhost_3000_account_oasis1qp25k3hv6ysrm6wntjnt6x0j5du0ud4qqchplrf2_stake (2)](https://user-images.githubusercontent.com/3758846/157794552-e270febd-7e97-473c-b9d7-b14c45621512.png) 

---

\* Active/inactive meaning [oasisscan/swagger.yml#L306-L309](https://github.com/oasisprotocol/oasis-wallet-web/blob/8c7b9d5/src/vendors/oasisscan/swagger.yml#L306-L309):
> If "true", an entity has a node that is registered for being a validator, node is up to date, and has successfully registered itself. However, it may or may not be part of validator set (top <scheduler.params.max_validators> by stake).

Reply:
While the warning does make things clearer, I am still torn on this because it still forces people that delegate to something not working, into the debonding period.  I am still of the opinion that instead of a warning this should just flat out reject the operation.

Inevitably someone will ignore the warning, and later complain once their tokens get stuck in debonding.  While it is on them, forcing people to wait till things recover (in what should be an increasingly rare occurence in the future), is likely better.

If people really can't wait to delegate the command line tools will always work...